src/lib/theme.js

```diff
@@ -69,8 +69,7 @@ const fonts = {
 };
 
 const config = {
-  initialColorMode: "light",
-  useSystemColorMode: true
+  initialColorMode: "system"
```

Comment:
```suggestion
  initialColorMode: "system",
  useSystemColorMode: true
```

I think we can just refer to [the first example in Chakra UI documentation](https://chakra-ui.com/docs/styled-system/color-mode#common-configurations), that is, our UI color mode changes with the system color mode. What do you think?

Reply:
FYI. [migration document](https://chakra-ui.com/getting-started/migration#breaking-changes)

Reply:
Sorry, you're right. I misunderstood the meaning of the document.

Reply:
Thanks for your information. I understood the related breaking change with color mode.

What I mean (and also the documentation means) is that our App's UI color mode *changes* with the system color mode.

---

[The *first example* in Chakra UI documentation](https://chakra-ui.com/docs/styled-system/color-mode#common-configurations):

```javascript
// A.
// System sets initial value.
// App subscribes to system color mode changes.
const config: ThemeConfig = {
  initialColorMode: 'system',
  useSystemColorMode: true,
}
```

---

**My understanding with a scene:**

It was 17:59 and the system color mode was `light`. Alice opened our App, whose color mode was also `light` as  expected.

Then, it was 18:00 and the system color mode changed to `dark`. Our App:

1. without `useSystemColorMode: true` (as the default value is `false`), our App's UI color mode remained `light`;
2. with `useSystemCOlorMode: true`, our App's UI color mode also changed to `dark`.

Reply:
For your information, the second example in Chakra UI documentation is in your commit:

```javascript
// B.
// System sets initial value.
// App color mode is detached from system color mode changes.
const config: ThemeConfig = {
  initialColorMode: 'system',
  useSystemColorMode: false,
}
```

Pay attention to the comments in the above.

Reply:
Got it. I've patched that, please review again.